Resources/Prototypes/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -65,6 +65,8 @@
     steps: 4
     zeroVisible: true
   - type: Appearance
+  - type: StaticPrice
+    price: 10000
```

Comment:
?

Reply:
> ?

I have set the price for the decartes pistol based on its description and rarity, I can lower the price to 7000 credits